dialect.go

```diff
@@ -8,47 +8,60 @@ import (
 // SQLDialect abstracts the details of specific SQL dialects
 // for goose's few SQL specific statements
 type SQLDialect interface {
-	createVersionTableSQL() string // sql string to create the db version table
-	insertVersionSQL() string      // sql string to insert the initial version table row
-	deleteVersionSQL() string      // sql string to delete version
-	dbVersionQuery(db *sql.DB) (*sql.Rows, error)
+	CreateVersionTableSQL() string // sql string to create the db version table
+	InsertVersionSQL() string      // sql string to insert the initial version table row
+	DeleteVersionSQL() string      // sql string to delete version
+	DBVersionQuery(db *sql.DB) (*sql.Rows, error)
```

Comment:
This is a breaking API change. We'll have to park this for now and consider it for goose v3.